types/mocha.d.ts

```diff
@@ -0,0 +1,214 @@
+// Type definitions for mocha 2.2.5
```

Comment:
I have not had much luck importing definition files via npm. Except for tools that are typescript 1st class citizens (e.g. RxJS but not things like mocha). 

DefinatelyTyped is a single **massive** repo of all definition files ever. You don't want to npm that things. The mocha file *is* direcly copied from DefinatelyTyped. 

To solve that problem. TSD is a manager wrapper for DefinitelyTypes that manually copies the type files to /typings. From TSD documentation https://github.com/DefinitelyTyped/tsd 
"It is recommended you check-in the definitions you install into your VCS."

So we could switch to use TSD for type definition management, but it would have no real impact on what gets checked into the repo if we follow its recommended practice. But it would add another package manager that needs installing which I think would be nicer to avoid.